google/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link
+      rel="icon"
+      type="image/svg+xml"
+      href="/vite.svg"
+    />
+    <link
+      rel="stylesheet"
+      href="https://unicons.iconscout.com/release/v4.0.8/css/line.css"
+    />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Vite App</title>
+  </head>
+  <body>
+    <!-- 
+      -header
+        -navgation
+          -gmail
+          -images
+          -icon
+          -user
+      -logo google
+      -main
+        -search
+          -icon
+          -search bar
+          -mic
+        -recommdation
+          -icon
+          -icon
+          -icon
+          -icon
+     -->
+    <div class="header">
+      <div>Gmail</div>
+      <div>Images</div>
+      <div>Icon</div>
+      <div>User</div>
+    </div>
+    <div class="con">
+      <div class="logo">Google</div>
+      <div class="main">
+        <div class="search">
+          <i class="uil uil-search"></i>
+
+          <input
+            type="text"
+            placeholder="Search Google or type a URL"
+          />
+
+          <i class="uil uil-microphone"></i>
+        </div>
+        <div class="rec">
```

Comment:
```suggestion
        <div class="recommendation">
```
 <div class="recommendation"> 应该放在<div class="search">里面，已保持宽度一致